project/LinkerdBuild.scala

```diff
@@ -529,6 +529,18 @@ object LinkerdBuild extends Base {
 
   // Unified documentation via the sbt-unidoc plugin
   val all = project("all", file("."))
-    .aggregate(k8s, consul, marathon, Linkerd.all, Namerd.all, Namerd.dcosBootstrap, Router.all, Namer.all, Interpreter.all, configCore, admin, testUtil)
+    .aggregate(
+      admin,
+      configCore,
+      consul,
+      k8s,
+      testUtil
+      marathon,
```

Comment:
Is this intended to be alphabetical?  In that case I'd put testUtil after marathon.